src/helpers/getSectionColors.ts

```diff
@@ -0,0 +1,20 @@
+/* Retrieves section color values.
+ *
+ * Requires a section type as defined by sectionDataMap. Accepts an optional color
+ * preference ("primary" or "secondary").
+ *
+ * @returns A string if one color is requested or an object containing both (default).
+ */
+import { sectionDataMap, SectionTypes } from "./types";
+
+export function getSectionColors(
+  type: SectionTypes,
+  colorVal?: "primary" | "secondary"
+) {
+  let colorName = sectionDataMap.filter((section) => section.type === type)[0]
```

Comment:
Removing the `[0]` breaks the function as it can't find the colorVals value.

As for the nit, I have found that using "color" as a prop name conflicts with Chakra's definition of the same prop. So I don't use it.

Reply:
I changed the look up table from an array to an object and adjusted the function accordingly. It would be possible to get rid of the function all together as the section colors are available with the passed `newsletterSignupType` prop as such: `sectionDataMap[newsletterSignupType]` and the color values with the `primary` or `secondary` key too. Let me know if that would be desirable.